cloud-gateway/src/main/java/com/registry/cloudgateway/CloudGatewayApplication.java

```diff
@@ -2,6 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
```

Comment:
This is an unused import, make sure to remove such imports. You can use IntelliJ to optimize your imports which rearranges them (alphabetically? not sure) and removes unused ones. The shortcut is Ctrl + Alt + O